### PR TITLE
Adwords migration handling

### PR DESCRIPTION
**Pull request by beevital**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/fivetran/terraform-provider-fivetran/compare/v0.4.5...HEAD)
+## [Unreleased](https://github.com/fivetran/terraform-provider-fivetran/compare/v0.4.6...HEAD)
+
+## [0.4.6](https://github.com/fivetran/terraform-provider-fivetran/compare/v0.4.5...v0.4.6) - 2022-06-15
+
+## Compatibility changes
+- Handle `adwords` service migration to `google_ads`
```
Comment (borisenko-i) [inline CHANGELOG.md:13]: Maybe give a bit more details on what's really changed and how it affects the users? Now it sounds like "we changed something related to adwords and google_ads"

File: fivetran/resource_connector.go [hunk 2]
```diff
@@ -388,7 +388,14 @@ func resourceConnectorCreate(ctx context.Context, d *schema.ResourceData, m inte
 	svc := client.NewConnectorCreate()
 
 	svc.GroupID(d.Get("group_id").(string))
-	svc.Service(d.Get("service").(string))
+
+	currentService := d.Get("service").(string)
+
+	if currentService == "adwords" {
+		return newDiagAppend(diags, diag.Error, "create error", "service `adwords` deprecated, use `google_ads instead")
```
Comment (borisenko-i) [inline fivetran/resource_connector.go:395]:
```suggestion
		return newDiagAppend(diags, diag.Error, "create error", "service `adwords` has been deprecated, use `google_ads` instead")
```

**Review by borisenko-i — APPROVED** [review]
TBH I have no idea if this is technically correct, but thanks for implementing 🙂

File: CHANGELOG.md (hunk 1 above)
Comment (beevital) [reply CHANGELOG.md:13]: We can write anything you propose to make it more clear

Comment (borisenko-i) [reply CHANGELOG.md:13]: I don't know, maybe something like
> Deprecate the `adwords` service in favor of the new `google_ads` service. All connector creation requests with the service `adwords` will now result in an error.

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,7 +5,15 @@ All notable changes to this project will be documented in this file.
...
-## [Unreleased](https://github.com/fivetran/terraform-provider-fivetran/compare/v0.4.5...HEAD)
+## [Unreleased](https://github.com/fivetran/terraform-provider-fivetran/compare/v0.4.6...HEAD)
+
+## [0.4.6](https://github.com/fivetran/terraform-provider-fivetran/compare/v0.4.5...v0.4.6) - 2022-06-15
+
+## Compatibility changes
+- Handle `adwords` service migration to `google_ads` for exisintg connectors. 
+- Deprecate the `adwords` service in favor of the new `google_ads` service. 
+
+NOT: All connector creation requests with the service `adwords` will now result in an error.
```
Comment (borisenko-i) [inline CHANGELOG.md:16]:
```suggestion
- Handle `adwords` service migration to `google_ads` for existing connectors. 
- Deprecate the `adwords` service in favor of the new `google_ads` service. 

NOTE: All connector creation requests with the service `adwords` will now result in an error.
```

**Review by Fivetran-RomanKolesnikov — APPROVED** [review]
Looks good!

**Pull request merged**